README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
For the distribution mode architecture description , can we describe like below, since if we use simulated hardware, we can simulate robot on different machine.

> Functions are distributed to different machines: Server, Client, Simulated robot(if needed).
> Client runs rviz to operate the robot and sensor, the robot and sensor can be real or simulated.
> Server runs functions to process point cloud and trajectory data.
> Simulated robot will run simulated robot node and point_cloud_generator(simulated sensor).
> Note: One server can support multiple clients.